CHANGELOG.asciidoc

```diff
@@ -52,6 +60,15 @@ https://github.com/elastic/beats/compare/v7.5.1...v7.6.0[View commits]
 - netflow: Fix bytes/packets counters on some devices (NSEL and Netstream). {pull}15449[15449]
 - netflow: Fix compatibility with some Cisco devices by changing the field `class_id` from short to long. {pull}15449[15449]
 - Fixed dashboard for Cisco ASA Firewall. {issue}15420[15420] {pull}15553[15553]
+- Fix s3 input hanging with GetObjectRequest API call by adding context_timeout config. {issue}15502[15502] {pull}15590[15590]
+- Add shared_credential_file to cloudtrail config {issue}15652[15652] {pull}15656[15656]
```

Comment:
is this a bug fix or an enhancement?

Reply:
Both above should be bug fixes.

Reply:
Based on the comment from @kaiyan-sheng , I'll leave these two items in the bug fix section.